script.js

```diff
@@ -1,4 +1,140 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
+/* Create a basic version of Scissors Paper Stone where the user inputs one of "scissors", "paper", or "stone", the program internally randomly chooses scissors, paper, or stone, and the program outputs whether the user won, the program won, or it's a draw.
+Rules: scissors beats paper, paper beats stone, and stone beats scissors. If both parties choose the same object, it's a draw. */
+
+// Add state to your program such that it keeps track of the number of times the user has won and the number of times the computer has won. Output this win-loss record in a format you like in the program output. You can also output the number of draws and/or each party's winning percentage if you'd like.
+
+// Ensuring the input is correct
```

Comment:
When submitting code, it is common practice to remove comments unless they directly explain the line of code below them. Something to remember for next time!